pkg/controller/direct/compute/forwardingrule_controller.go

```diff
@@ -282,7 +283,13 @@ func (a *forwardingRuleAdapter) Create(ctx context.Context, createOp *directbase
 		return mapCtx.Err()
 	}
 	forwardingRule.Name = direct.LazyPtr(a.id.forwardingRule)
-	forwardingRule.Labels = desired.Labels
+	target := direct.ValueOf(forwardingRule.Target)
+
+	// API restriction: Labels are invalid in Private Service Connect Forwarding Rule.
+	// TF workaround: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/944
```

Comment:
mind updating the comment? "TF workaround" --> "ConfigConnector workaround on TF-based resource"